### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by Max-Tweddell**
#4

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-import WaitIndicator from './WaitIndicator'
+export default class App extends React.Component {
+  constructor (props) {
+    super(props)
+    // TODO state and logic stuff
+  }
+  render () {
+    return (
+      <div className='app'>
+        <MessageList data={this.state.messageData} />
```
Comment (don-smith) [inline client/components/App.jsx]: Let's change the `data` prop to `messages`.

File: client/components/MessageList.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Message from './Message'
+
+export default function MessageList (props) {
+  return (
+    props.data.map(messageData => {
```
Comment (don-smith) [inline client/components/MessageList.jsx]: Let's change this to `props.messages.map(message => {`

File: client/components/Person.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+let styles = {
```
Comment (don-smith) [inline client/components/Person.jsx]: `let` ➡️ `const`

File: client/components/Message.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Person from './Person'
+
+export default function Message (props) {
+  return (
+    <div>
+      <Person imageData={props.message.sender}/>
+      <img src={props.message.src}/>
```
Comment (don-smith) [inline client/components/Message.jsx]: Shouldn't this be `props.message.imageUrl`?

File: client/components/Message.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Person from './Person'
+
+export default function Message (props) {
+  return (
+    <div>
+      <Person imageData={props.message.sender}/>
```
Comment (don-smith) [inline client/components/Message.jsx]: Let's change this `imageData` prop to `person`.

File: client/components/Person.jsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  borderRadius: "150px",
+  background: "white",
+  width: "300px",
+  height: "300px"
+}
+
+export default function Person (props) {
+  return (
+    <img style={styles} src={props.imageData.url} alt={props.imageData.name}/>
```
Comment (don-smith) [inline client/components/Person.jsx]: This will turn into `src={props.person.photo}`

**Comment by Max-Tweddell** [comment]
Okay, those problems should be fixed

**Pull request merged**